src/gatsby-node.js

```diff
@@ -19,8 +17,9 @@ export const onCreateWebpackConfig = ({ actions, stage }) => {
     actions.setWebpackConfig({
       plugins: [
         new LoadablePlugin({
-          filename: (isWin ? loadableStatsName : statsPath),
+          filename: statsPath,
           writeToDisk: true,
+          outputAsset: !isWin
```

Comment:
Where does Windows write the file to?  I may want to include more details on Windows behavior in the docs.

Reply:
Before the fix it was writing in the root directory, now it should write it in the `.cache` directory.

The fix was taken from https://github.com/graysonhicks/gatsby-plugin-loadable-components-ssr/issues/25#issuecomment-865082674 workaround.